tests/ifs/ifs.c

```diff
@@ -82,8 +88,74 @@ static ssize_t read_file(int dfd, const char *filename, char buf[static restrict
         return n;
 }
 
+static bool load_test_file(int dfd, struct test *test, ifs_test_t *ifs_info)
+{
+    char current_buf[BUFLEN], status_buf[BUFLEN];
+    int next_test, current_test;
+
+    /* read both files status and current_batch */
+    read_file(dfd, "status", status_buf);
+    read_file(dfd, "current_batch", current_buf);
+
+    /* when previous run has a status of fail, skip test */
+    if (memcmp(status_buf, "fail", strlen("fail")) == 0)
```

Comment:
I wonder: should we have a test knob to force running?

Reply:
I think we should, do we have a get_testspecific_knob_value_bool() by any chance? Would be useful only add a "force" option

Reply:
We can either add that or just use integers. `force=1`

Option only allowed if `$USER == "luke"` :-)